generate_config.py

```diff
@@ -0,0 +1,91 @@
+import json
```

Comment:
Can we call this file `generate_datasource_column_config.py`? Just to be more explicit.

Reply:
I'll name them `datasources_column_config_generate` and `datasources_column_config_merge`.  That gets to what you want and will put those 2 scripts next to each other in the files.